drape_frontend/stylist.cpp

```diff
@@ -103,19 +103,19 @@ class Aggregator
 private:
   void ProcessKey(drule::Key const & key)
   {
-    double depth = key.m_priority;
-    if (m_depthLayer != feature::LAYER_EMPTY)
+    float depth = key.m_priority;
```

Comment:
Leave double here to keep precise calculations when multiply or some.

Reply:
Ok, but do we really need extra precision here?
	
All of the operands are whole numbers, so the remainder of the division will be a whole number too.
TBH all of this could be done with integers, maybe it'll be better to make m_depth `int` instead of float and cast to float only before passing it to e.g. `glsl::vec3`?
What do you think?